stake-pool/cli/src/main.rs

```diff
@@ -719,6 +817,67 @@ fn main() {
                     .help("Stake account to receive SOL from the stake pool. Defaults to a new stake account."),
             )
         )
+        .subcommand(SubCommand::with_name("set-staking-auth").about("Changes staking authority of one of the accounts from the stake pool.")
+            .arg(
+                Arg::with_name("pool")
+                    .long("pool")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .required(true)
+                    .help("Stake pool address."),
+            )
+            .arg(
+                Arg::with_name("stake_account")
+                    .long("stake-account")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .required(true)
+                    .help("Stake account address to change staking authority."),
+            )
+            .arg(
+                Arg::with_name("new_staker")
+                    .long("new-staker")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .required(true)
+                    .help("Public key of the new staker account."),
+            )
+        )
+        .subcommand(SubCommand::with_name("set-owner").about("Changes owner or fee receiver account for the stake pool.")
+            .arg(
+                Arg::with_name("pool")
+                    .long("pool")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .required(true)
+                    .help("Stake pool address."),
+            )
+            .arg(
+                Arg::with_name("new_owner")
+                    .long("new-owner")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .help("Public key for the new stake pool owner."),
+            )
+            .arg(
+                Arg::with_name("new_fee_receiver")
+                    .long("new-fee-receiver")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .help("Public key for the new account to set as the stake pool fee receiver."),
+            )
+            .group(ArgGroup::with_name("new_accounts")
+                .arg("new_owner")
+                .arg("new_fee_receiver")
+                .required(true)
+            )
```

Comment:
In case someone wants to update both at once, this should include `multiple(true)`.  If someone wants to reassign both, we have a footgun if they do it in the wrong order, e.g.:

case 1:
* reassign fee account -> succeeds
* reassign owner -> succeeds

case 2:
* reassign owner -> succeeds
* reassign fee account -> fails, no longer the owner

So let's have both be possible at once.